src/settings.ts

```diff
@@ -97,6 +97,7 @@ export const SPARKLEVERSE_PHOTOBOOTH_URL =
   "https://burn.sparklever.se/in/outsnappedphotoboothcamp";
 export const PLAYA_LOCATION_NAME = "playa";
 export const PLAYA_TIMEZONE = "America/Los_Angeles";
+export const DOWNLOAD_CHROME_URL = "https://www.google.ru/chrome/";
```

Comment:
```suggestion
export const DOWNLOAD_CHROME_URL = "https://www.google.com/chrome/";
```

I assume using the general link for downloading Chrome is better